clang/test/CIR/CodeGen/tbaa3.c

```diff
@@ -0,0 +1,79 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir -O1
+// RUN: FileCheck --check-prefix=CIR --input-file=%t.cir %s
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o %t.ll -O1
+// RUN: FileCheck --check-prefix=LLVM --input-file=%t.ll %s
+
+// CIR: #tbaa_type_desc = #llvm.tbaa_type_desc<id = "omnipotent char", members = {<#tbaa_root, 0>}>
+// CIR: #[[INT_DESC:.*]] = #llvm.tbaa_type_desc<id = "int", members = {<#tbaa_type_desc, 0>}>
+// CIR: #[[ANY_POINTER:.*]] = #llvm.tbaa_type_desc<id = "any pointer", members = {<#tbaa_type_desc, 0>}>
+// CIR: #[[TBAA_TAG_INT:.*]] = #llvm.tbaa_tag<base_type = #[[INT_DESC]], access_type = #[[INT_DESC]], offset = 0>
+// CIR: #[[TBAA_TAG_PTR:.*]] = #llvm.tbaa_tag<base_type = #[[ANY_POINTER]], access_type = #[[ANY_POINTER]], offset = 0>
+// CIR: #[[relation_t_DESC:.*]] = #llvm.tbaa_type_desc<id = "relation_t", members = {<#[[ANY_POINTER]], 0>, <#[[INT_DESC]], 8>}>
+// CIR: #[[TBAA_TAG_relation_t_int:.*]] = #llvm.tbaa_tag<base_type = #[[relation_t_DESC]], access_type = #[[INT_DESC]], offset = 8>
+// CIR: #[[TBAA_TAG_relation_t_pointer:.*]] = #llvm.tbaa_tag<base_type = #[[relation_t_DESC]], access_type = #[[ANY_POINTER]], offset = 0>
+typedef int intkey_t;
+typedef int value_t;
+
+typedef struct tuple_t    tuple_t;
+typedef struct relation_t relation_t;
+
+struct tuple_t {
+    intkey_t key;
+    value_t  payload;
```

Comment:
I'm fine with any indentation on tests, but they should be consistent with one file, looks like this differs from the next struct